### PR TITLE
removed current unecessary items, started building base text/functionality

### PR DESCRIPTION
**Pull request by GabriellaMSanchez**
Removed base create-react-app items that were not necessary at this time (per online course). Began building base trail-finder text and functionality.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    return (
+        <div>
+            <h1>Trail Finder</h1>
+            <h4>{new Date().toLocaleDateString()}</h4>
+            <textarea
```
Comment (DanielRyanSmith) [inline src/App.js:6]: This should probably be an `input` element. Text areas are typically used for writing lots of text.

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,13 @@
...
 
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+// 3) tell react to take control of that element
+const root = ReactDOM.createRoot(el);
+
+// 4) show the component on the screen
+root.render(<App />);
```
Comment (DanielRyanSmith) [inline src/index.js:13]: New line at end of file.

**Review by DanielRyanSmith — CHANGES_REQUESTED** [review]
Deleting the favicon at `public/favicon.ico` means that there is no longer an image that displays on the tab in the browser. Is this intended with this change?

File: src/index.js (hunk 2 above)
Comment (GabriellaMSanchez) [reply src/index.js:13]: fixed! :)

File: src/App.js (hunk 1 above)
Comment (GabriellaMSanchez) [reply src/App.js:6]: fixed! :)

**Review by DanielRyanSmith — APPROVED** [review]
Nice 😎

**Pull request merged**